### ISSUE TITLE
PlayerMock#chat does not triggers AsyncPlayerChatEvent.

### ISSUE THREAD

**Issue by Plasticable** [seeseemelk/MockBukkit#5]
Hi. I want to test my plugin that working with chat.
In my plugin i created and registered listener that listens AsyncPlayerChatEvent.
When i call in tests
```
PlayerMock spammer = server.addPlayer();
spammer.chat("Some message");
```
nothing happens.  
I also tried to simulate block break and listen corresponding event, it works, but chat event doesn't.
Code from my event listener class:
```
@EventHandler
public void onChat(AsyncPlayerChatEvent event) {
    System.out.println(event.getEventName());
}

@EventHandler
public void onBlockBreak(BlockBreakEvent event) {
    System.out.println(event.getEventName());
}
```

MockBukkit dependency: `com.github.seeseemelk:MockBukkit:-SNAPSHOT`

**Comment by seeseemelk** [comment]
For the block breaking, have you made sure the player is not in creative mode? When in creative mode, a block break event is never fired. (If it still doesn't happen in survival mode then there is a real issue somewhere).

And about the player chat, AsyncPlayerChatEvent is not yet implemented. I'm not sure if PlayerChatEvent is, but I'll try to work on that when I find the time.

**Comment by seeseemelk** [comment]
I've looked at it for a bit but for me the BlockBreakEvent seems to be working fine. Are you sure you've correctly registered your event listener using `Bukkit.getPluginManager.registerEvents()`? You need to do this even for the main class.

**Comment by seeseemelk** [comment]
I renamed the branches so it's a lot easier to develop for several different versions of Bukkit. You therefore want to change your dependency to com.github.seeseemelk:MockBukkit:v1.12-SNAPSHOT. Sorry for this inconvenience.

**Comment by Plasticable** [comment]
You misunderstood me. Breaking block test works absolutely fine, i mentioned about it to show that my listener which listens both BlockBreakEvent and AsyncPlayerChatEvent is registered. 
I also have tried listen PlayerChatEvent, and anyway it's not triggered.
It seems that PlayerMock#chat is really not implemented but it didn't throw NotImplementedException.

**Comment by seeseemelk** [comment]
I'm looking through the repository and it seems that I did forgot to make it throw an UnimplementedOperationException. It should now be implemented in the branches v1.13 and develop-1.12. Let me know if it works so that I can merge develop-1.12 into v1.12
